crates/ty_python_semantic/src/semantic_index/use_def/place_state.rs

```diff
@@ -71,16 +71,12 @@ impl ScopedDefinitionId {
     }
 }
 
-/// Can keep inline this many live bindings or declarations per place at a given time; more will
-/// go to heap.
-const INLINE_DEFINITIONS_PER_PLACE: usize = 4;
-
```

Comment:
Why remove this constant? It's used in two places, and it seems strictly better to have a documented constant rather than an arbitrary magic number in the code.